src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1082,13 +1371,13 @@ Prevent Long URL From Breaking Out of Container
 */
 .emailLog-text {
     -ms-word-break: break-all;
```

Comment:
This is still in the wrong position. There is a (in fact, two?) `word-break` rule(s) down there.

Reply:
`word-break: break-all;` is standard while `word-break: break-word;` is not standard and is defined for WebKit.